lib/cc/analyzer/issue_location_format_validation.rb

```diff
@@ -30,13 +30,13 @@ def valid_positions?(positions)
       def valid_position?(position)
         position &&
           (
-            (position["line"] && position["column"]) ||
-            position["offset"]
+            [position["line"], position["column"]].all? { |value| value.is_a?(Integer) } ||
+            position["offset"] && position["offset"].is_a?(Integer)
```

Comment:
I don't think it's better to cast in this case because any string could be cast to a `0`. You could have `"offset": "ohmygod"` and still get a `0` with `to_i`. 

The bugsnag error I saw actually pointed to a bug in the radon engine. I don't think we should have generated this issue at all and it's the only issue I saw with a string location value.

```json
{
    "categories": [
        "Bug Risk"
    ],
    "check_name": "Complexity",
    "content": {
        "body": "We encountered an error attempting to analyze this line."
    },
    "description": "Error: invalid syntax (<unknown>, line 102)",
    "fingerprint": "d2fec2141485c30d047b12f9f06283ee",
    "location": {
        "lines": {
            "begin": "102",
            "end": "102"
        },
        "path": "packages/grid_control_gui/xmpp/features.py"
    },
    "remediation_points": 1000000,
    "type": "issue"
}
```